__DEFINES/_macros.dm

```diff
@@ -162,6 +162,8 @@
 
 #define isfood(A) istype(A, /obj/item/weapon/reagent_containers/food)
 
+#define ismuzzle(A) A.is_muzzle
```

Comment:
weird choice to make a define for this; you've netted no savings in terms of characters and since this isn't a proc it's not like you can override it; this seems unnecessary but you don't need to get rid of it either